x-pack/packages/security-solution/distribution_bar/src/distribution_bar.tsx

```diff
@@ -155,7 +163,13 @@ export const DistributionBar: React.FC<DistributionBarProps> = React.memo(functi
     const prettyNumber = numeral(stat.count).format('0,0a');
 
     return (
-      <div key={stat.key} css={partStyle} data-test-subj={`${dataTestSubj}__part`}>
+      // eslint-disable-next-line jsx-a11y/click-events-have-key-events
```

Comment:
The reason why have this a11y rule failing is because we have a click event on a `div`
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/click-events-have-key-events.md#fail

The solution might be to add this,  the important thing is to add the role.
```
      onClick={stat.onClick}
      onKeyDown={(event) => {
           if (event.key === 'Enter' || event.key === ' ') {
             stat.onClick?.();
           }
         }}
         tabIndex={0}
       role="button"
```